pythonAPIClient/client.py

```diff
@@ -296,13 +296,170 @@ def usage_by_client(self, clientID, owner):
                 raise InternalError()
         return response.json()['api_usage']
 
-    # TODO: add a camera to database
-    def add_camera(self, camera):
-        pass
+    def add_camera(self, camera_type, is_active_image, is_active_video, snapshot_url, m3u8_url,
+                   ip=None, legacy_cameraID=None, source=None, latitude=None, longitude=None,
+                   country=None, state=None, city=None, resolution_width=None,
+                   resolution_height=None, utc_offset=None, timezone_id=None, timezone_name=None,
+                   reference_logo=None, reference_url=None, port=None, brand=None, model=None,
+                   image_path=None, video_path=None):
+
+        """add_camera initialization method.
+
+                Parameters
+                ----------
+                camera_type : str
+                    Type of camera.
+                    Allowed values: 'ip', 'non_ip', 'stream'/
+                is_active_image : bool
+                    If the camera is active and can get images.
+                    This field can identify true/false case-insensitively and 0/1.
+                is_active_video : bool
+                    If the camera is active and can get video.
+                    This field can identify true/false case-insensitively and 0/1.
+                ip : str
+                    (ip_camera) IP address of the camera.
+                snapshot_url : str
+                    (non_ip_camera) Url to retrieve snapshots from the camera.
+                m3u8_url : str
+                    (stream_camera) Url to retrieve stream from the camera.
+                legacy_cameraID : num
```

Comment:
type should be int for this one. also, for those optional parameters, place an optional flag like `legacy_cameraID : int, optional`